data/campaigns/Wings_of_Victory/scenarios/05_Islands.cfg

```diff
@@ -0,0 +1,336 @@
+#textdomain wesnoth-wov
+# This scenario is intended to be somewhat difficult to convey the dangers of the journey, and to drain gold.
+[scenario]
+    id=05_Islands
+    name= _ "Islands"
+    next_scenario=06_The_Three_Sisters
+    {WOV_MAP 05_Islands.map} # TRoW Rough Landing map, unchanged.
+    {TURNS 26 26 24}
+    {SUMMER_SCHEDULE_DUSK}
+    carryover_percentage=40
+    carryover_add=yes
+
+    {INTRO_AND_SCENARIO_MUSIC revelation.ogg wanderer.ogg}
+    {EXTRA_SCENARIO_MUSIC traveling_minstrels.ogg}
+    {EXTRA_SCENARIO_MUSIC knolls.ogg}
+
+    [story]
+        [part]
+            story= _ "They had little time to prepare the new flight. The new hatchlings could not go long without food...but worse, if the breeders grew heavy with eggs they would lose their ability to fly."
+        [/part]
+        [part]
+            story= _ "Galun prepared for true long-distance travel by abandoning the massive armor of the Clashers. The flight took only the lighter tools with it, counting on re-creating the heavier ones in the new land. Instead, they loaded up with sling bags of dried meat. That, and the prey they could take from the Great Ocean on their journey, might see them through."
+        [/part]
+        [part]
+            story= _ "All the drakes knew that if Galun was successful in finding new land, their people would have new hope. The Flights of Verkon and Kerath sent Sky Drakes to assist Galun in finding a way to the Great Continent, and to report back when it was located."
+        [/part]
+        [part]
+            # TODO: where did they sleep?
+            story= _ "For many days they flew over unknown waters, growing weary and hungry and wing-sore. Then Galun, scouting ahead with a few followers, sighted land..."
+        [/part]
+    [/story]
+
+    # wmllint: validate-off
+    [side]
+        side=1
+        controller=human
+        recruit=Drake Burner, Drake Fighter, Drake Glider
+        gold=200
+        save_id=Player
+        team_name=hero
+        user_team_name= _ "Drakes"
+        {FLAG_VARIANT long}
+        [leader]
+            {GALUN}
+        [/leader]
+        [unit]
+            {VANK}
+            placement=leader
+        [/unit]
+    [/side]
```

Comment:
When I load the S5 turn 1 autosave, it says "It is now josteph’s turn", whereas for say HttT it says "It is now Konrad’s turn", so I expected it to say "It is now Galun’s turn" here. I guess `side_name=_ "Galun"` should be added to the [side] tag?

Or maybe the engine should always try a [side][leader]name= key if there's no [side]name= key.

Reply:
Yeah, the engine should probably do that.

Reply:
Noted